### ISSUE TITLE
Nav buttons adjustment

### ISSUE THREAD

**Issue by MattCSmith** [zeroDevs/coding_challenge-13#128]
Perhaps we could try and give the navbar buttons the same transparent background color the nav tabs have, and see how it looks:

![image](https://user-images.githubusercontent.com/6190356/51323560-772aff80-1a60-11e9-8189-ca608ba7f783.png)

**Comment by luxumbra** [comment]
Hi Matt, I'd worry that the lack of contrast on these buttons would be detrimental to a11y - I feel that the pink of the text in the current nav buttons is too close to some of the background.  I don't know what others think?

**Comment by MattCSmith** [comment]
This is a good point. Do you think making the background of the button more white would solve that problem?

**Comment by phillpeters** [comment]
Based on your comments, I made a small change.  Changing the text color of the nav buttons to red as @Dromediansk suggested increases the contrast of the text against the background, but is also consistent with the nav tabs.  In my opinion this looks slightly better than adding the semi-transparent white background.
![screenshot from 2019-01-18 14-33-52](https://user-images.githubusercontent.com/34535492/51414008-1c5fd980-1b2e-11e9-8db6-8e3d553c4450.png)

**Comment by MattCSmith** [comment]
Hmm I'm undecided now on whether the `newsletter` and `data` buttons should have a transparent background. 

What do you think @aneagoie ?

**Comment by aneagoie** [comment]
I agree with @phillpeters 👍

**Comment by MattCSmith** [comment]
Closing as this has been resolved!

**Comment by The24thDS** [comment]
@all-contributors add @Dromediansk for question

**Comment by The24thDS** [comment]
@all-contributors add @phillpeters for question, code and ideas

**Comment by The24thDS** [comment]
@all-contributors add @luxumbra for ideas
